source/fab/steps/compile_fortran.py

```diff
@@ -65,14 +65,17 @@ def __init__(self, compiler: str = None, common_flags: List[str] = None, path_fl
         compiler = compiler or os.getenv('FC', 'gfortran -c')
         common_flags = common_flags or []
         env_flags = os.getenv('FFLAGS', '').split()
-        self.exe = compiler or os.getenv('FC', 'gfortran -c')
+        self.exe: str = compiler or os.getenv('FC', 'gfortran -c')  # type: ignore
```

Comment:
My guess is that mypy doesn't understand that the return value of `getenv` will never be `None` in this case. I think it just knows it can be `None` sometimes.